### ISSUE TITLE
[NVDec] failed to create cuvid decoder CUDA_ERROR_NO_DEVICE

### ISSUE THREAD

**Issue by ignacioalpiste** [gpac/gpac#898]
Hello everyone,
I am trying to see a video encoded in hevc:
> MP4Box -v -add Cactus_1920x1080_50_qp32.bin:FMT=HEVC -fps 50 -new output.mp4
> MP4Client output.mp4

The window appears but with no video, black and in the console appears the next problem:
[NVDec] failed to create cuvid decoder CUDA_ERROR_NO_DEVICE

I have a graphic card that uses the hardware to decode hevc but it does not work.

Any solution?

Thanks, Ignacio.

**Comment by aureliendavid** [comment]
Hi. 

Can you post the video (or a sample) somewhere so we can test?

(you can drop files here for example: https://www.mediafire.com/filedrop/filedrop_hosted.php?drop=eec9e058a9486fe4e99c33021481d9e1826ca9dbc242a6cfaab0fe95da5e5d95)

thanks

**Comment by ChangJongHyun** [comment]
I have same problem ㅠ.ㅠ

**Comment by ChangJongHyun** [comment]
I have same problem.

I'm trying to see tiled streaming example on:

> MP4Client http://download.tsi.telecom-paristech.fr/gpac/SRD/srd_hevc/multi_rate/hevc_srd.mpd

I got mpd url to

https://gpac.wp.imt.fr/2017/02/01/hevc-tile-based-adaptation-guide/

here is my error log.

![zd](https://user-images.githubusercontent.com/26322536/40060174-f87acf8a-5890-11e8-9636-11b73dc5a50e.PNG)

I used gpac version 0.72

and I installed cuda v9.0.

How can i solve this problem?

thanks
